front/CODING_RULES.md

```diff
@@ -186,6 +186,20 @@ id: ModelId;
 }
 ```
 
+### [BACK11] Resource invariant: Use "Model" suffix for Sequelize models when creating Resources
+
+When creating a new Resource that wraps a Sequelize model, the model should be renamed to include the "Model" suffix for clarity (e.g., `Conversation` becomes `ConversationModel`). This naming convention helps distinguish between the Resource interface and the underlying Sequelize model implementation.
```

Comment:
The file is wrapped at 100 column as per CODING_RULE GEN6 :p

Reply:
I should've them tattoed